.jscsrc

```diff
@@ -1,4 +1,10 @@
 {
   "preset": "airbnb",
-  "excludeFiles": ['dist']
+  "excludeFiles": ['dist'],
+  "requireSpacesInsideObjectBrackets": false,
+  "requireCamelCaseOrUpperCaseIdentifiers": {
+    allExcept: [
+      "child_process",
```

Comment:
Yeah, annoyingly eslint and friends seem to use weird pseudo-JSON. I think it's mainly so you can have things like comments etc.